repository/c.json

```diff
@@ -3604,8 +3604,10 @@
 			]
 		},
 		{
-			"name": "Cool & Clear",
+			"name": "Cool & Clear - Color Scheme",
+			"previous_names": ["Cool & Clear"],
 			"details": "https://github.com/Doodpants/Cool-and-Clear-Color-Scheme",
+			"description": "A tastefully minimal, high contrast, light-mode color scheme",
```

Comment:
We advocate keeping the description out of the repo here, so that we don't need to handle an PR every time someone wants to change it. Also in this case, sure you can strip "for Sublime Text" in this context, but it's not really a problem to leave it in.